client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/simulering/SimuleringResponseMapper.kt

```diff
@@ -0,0 +1,86 @@
+package no.nav.su.se.bakover.client.oppdrag.simulering
+
+import no.nav.su.se.bakover.domain.Fnr
+import no.nav.su.se.bakover.domain.oppdrag.Utbetaling
+import no.nav.su.se.bakover.domain.oppdrag.simulering.KlasseKode
+import no.nav.su.se.bakover.domain.oppdrag.simulering.KlasseType
+import no.nav.su.se.bakover.domain.oppdrag.simulering.Simulering
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimulertDetaljer
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimulertPeriode
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimulertUtbetaling
+import no.nav.system.os.entiteter.beregningskjema.BeregningStoppnivaa
+import no.nav.system.os.entiteter.beregningskjema.BeregningStoppnivaaDetaljer
+import no.nav.system.os.entiteter.beregningskjema.BeregningsPeriode
+import no.nav.system.os.tjenester.simulerfpservice.simulerfpserviceservicetypes.SimulerBeregningResponse
+import java.time.LocalDate
+
+internal class SimuleringResponseMapper private constructor(
+    val simulering: Simulering,
+) {
+    constructor(oppdragResponse: SimulerBeregningResponse) : this(oppdragResponse.map())
+    constructor(utbetaling: Utbetaling) : this(utbetaling.mapTomResponsFraOppdrag())
+}
+
+private fun SimulerBeregningResponse.map() =
+    Simulering(
+        gjelderId = Fnr(simulering.gjelderId),
+        gjelderNavn = simulering.gjelderNavn.trim(),
+        datoBeregnet = LocalDate.parse(simulering.datoBeregnet),
+        nettoBeløp = simulering.belop.toInt(),
+        periodeList = simulering.beregningsPeriode.map { it.map() },
```

Comment:
Jeg foretrekker at disse extension-metodene heller har navn som `tilX`. `map` her blir forvirrende når man er vant med f.eks. `Either.map`, `List.map` osv, pluss at linjer som akkurat denne ser rare ut. Jeg syns det leser mye bedre med

```suggestion
        periodeList = simulering.beregningsPeriode.map { it.toSimulertPeriode() },
```

Reply:
Can do.